src/main/java/seedu/staffsnap/model/interview/Interview.java

```diff
@@ -48,19 +60,30 @@ public boolean equals(Object other) {
         }
 
         Interview otherInterview = (Interview) other;
-        return type.equals(otherInterview.type);
+        return type.equals(otherInterview.type)
+                && rating.equals(otherInterview.rating);
+    }
+
+    /**
+     * Returns true if the applicant has an interview with the same interview type.
+     */
+    public boolean isSameInterview(List<Interview> otherInterviews) {
+        return otherInterviews.stream().anyMatch(interview -> interview.getType().equals(getType()));
```

Comment:
`isSameInterview` seems to imply we're comparing an instance of Interview with another instance of Interview. 

Perhaps we can rename it has `isContainedIn` to better reflect the method signature? Any better name suggestions? 😆

Reply:
That's a very good suggestion 😄